static/app/components/replays/replayPlayer.tsx

```diff
@@ -75,7 +75,7 @@ function BasePlayerRoot({className}: Props) {
 
   return (
     <Panel isFullscreen={isFullscreen}>
-      <Centered ref={windowEl} data-test-id="replay-window">
+      <Centered ref={windowEl} className="rr-block" data-test-id="replay-window">
```

Comment:
In our SDK we're using `sr-block`

```suggestion
      <Centered ref={windowEl} className="sr-block" data-test-id="replay-window">
```